system/requirements-ubuntu-2004.txt

```diff
@@ -5,4 +5,7 @@ testinfra==1.14.1
 python3-indy==1.16.0
 docker==3.7.0
 more-itertools==8.10.0
-async_generator
\ No newline at end of file
+async_generator
+indy_vdr~=0.3.4
```

Comment:
@MeltedGlaciers Let's use == instead of ~=, unexpected package upgrade may break something.

Reply:
Fixed